docs/platforms/android/Kotlin-formatting.md

```diff
@@ -0,0 +1,27 @@
+# Setting up Kotlin formatting in Android Studio
+
+Kotlin code in this repository is formatted and linted with [`ktlint`](https://github.com/pinterest/ktlint).
+If you
+1. have submitted Kotlin code only to learn this when the analyzer check fails
+2. use Android Studio
+
+
+then there is good news! Android Studio can be configured to use `ktlint` to automatically apply formatting and highlight issues. To do this:
+
+
+1. Install the `ktlint` extension for Android Studio
+
+   a. On Mac, this is `Android Studio > Settings > Plugins > ` Search for `ktlint`.
+
+2. Set the ruleset to be the same as the version used in `ci.yaml` (as of writing this is 1.1.1), and the baseline to be `dev/bots/test/analyze-test-input/ktlint-baseline.xml`.
```

Comment:
make ci.yaml a hyperlink or file link or at minimum `<flutter framework>/.ci.yaml` Note it is dot ci dot yaml.

Reply:
Changed it to a link format that appears to work on github